lib/params/ParamsMgr.cpp

```diff
@@ -1312,7 +1312,8 @@ void ParamsMgr::PMgrStateSave::EndGroup()
 
     if (!GetEnabled()) return;
 
-    assert(_groups.size());
+    // If we are going to return in the next statement, why also assert for the same condition?
+    //assert(_groups.size());
```

Comment:
This assert needs to remain here. I think the intention was to have it be a VAssert so that it would only crash in debug builds and silently fail on release builds.

Reply:
I'm seeing this happen in other places in the code, so I'll file issue #3717.